ts/features/wallet/bancomat/screen/BancomatDetailScreen.tsx

```diff
@@ -62,13 +62,11 @@ const UnsubscribeButton = (props: { onPress?: () => void }) => (
  */
 const startCoBadge = (props: Props) => {
   const bancomat = props.navigation.getParam("bancomat");
-  if (bancomat.abiInfo?.abi) {
-    props.addCoBadge(bancomat.abiInfo.abi);
-  } else {
+  if (bancomat.info.issuerAbiCode === undefined) {
     showToast(I18n.t("global.genericError"), "danger");
-    void mixpanelTrack("BANCOMAT_DETAIL_NO_ABI_ERROR", {
-      issuerAbiCode: bancomat.info.issuerAbiCode
-    });
+    void mixpanelTrack("BANCOMAT_DETAIL_NO_ABI_ERROR");
```

Comment:
```suggestion
```
The scope of this action is track when an user have a bancomat with an ABI and that ABI is not in the remote ABI list.
With the new implementation, I think this tracking action should be moved in `CoBadgeChosenBankScreen` (see next comment)

Reply:
I leaved this tracked event because notify if there is some unexpected problem with the received data.